config/default/config_split.config_split.admissions.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 1aa7f804-a75a-4201-bb26-1a50922ec5dd
+langcode: en
+status: true
```

Comment:
Running `blt drupal:install` locally, I'm seeing the following error:
```
<li>The configuration synchronization failed validation.</li>
                  <li>Unable to install the <em class="placeholder">admissions_core</em> module since it does not exist.</li>
```

I think this is due to the admissions split being exported to the default config directory and enabled.

Reply:
good catch. i was treating it like a feature split...